### PR TITLE
[SUB-TASK][KPIP-4] Refine the batch response and render

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

This is make some enhancements for batch response:
- combine batch name and batch app name
- flatten the batchInfo map to `appId`, `appUrl`, `appState` and`appDiagnostic`

Using https://github.com/JakeWharton/flip-tables for table render, which support that the column value could be a multiple-line string.
### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

Batch List:
image

Batch Report:
image

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java [hunk 1]
```diff
@@ -17,16 +17,17 @@
...
 public class Batch {
   private String id;
   private String user;
   private String batchType;
   private String name;
-  private Map<String, String> batchInfo;
+  private String appId;
+  private String appUrl;
+  private String appState;
+  private String appDx;
```
Comment (turboFei) [inline kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java:30]: DX is the abbreviation of diagnosis

**Comment by turboFei** [comment]
```
error: failed to solve: openjdk:8-jdk: failed to do request: Head "https://registry-1.docker.io/v2/library/openjdk/manifests/8-jdk": net/http: TLS handshake timeout
Error: buildx failed with: error: failed to solve: openjdk:8-jdk: failed to do request: Head "https://registry-1.docker.io/v2/library/openjdk/manifests/8-jdk": net/http: TLS handshake timeout
```

GA failure does not relate with this pr.

**Comment by turboFei** [comment]
also cc @lightning-L

File: dev/dependencyList [hunk 2]
```diff
@@ -27,6 +27,7 @@ curator-client/2.12.0//curator-client-2.12.0.jar
 curator-framework/2.12.0//curator-framework-2.12.0.jar
 curator-recipes/2.12.0//curator-recipes-2.12.0.jar
 derby/10.14.2.0//derby-10.14.2.0.jar
+fliptables/1.0.2//fliptables-1.0.2.jar
```
[the remark this answers is not in the document]
Comment (turboFei) [reply dev/dependencyList:30]: added

**Review by ulysses-you — APPROVED** [review]

**Pull request closed without merging**